input/pagecontent/col.md

```diff
@@ -16,21 +16,21 @@ In the case of colorectal cancer screening, two types of data are reported: 1) t
 
 |Resource Type|Profile Name|Link to Profile|
 |---|---|---|
-|Condition|QI Core Condition Problems Health Concerns Profile|[QI Core Condition Problems Health Concerns]|
-|Coverage|QI Core Coverage Profile|[QI Core Coverage]|
-|DiagnosticReport|QI Core DiagnosticReport Profile|[QI Core DiagnosticReport]|
-|Encounter|QI Core Encounter Profile|[QI Core Encounter]|
+|Condition|QI-Core Condition Problems Health Concerns Profile|[QI-Core Condition Problems Health Concerns]|
```

Comment:
For these profile names and links to profiles, do not change QI Core to QI-Core, because the actual profile name has it as "QICore" without a space in between or dash, such as "QICore Encounter". I think it is better to display them to match the profile names.